vanilla-framework/vanilla-brochure-theme
# Add muted heading pattern

## Done

Add muted heading pattern

## QA

- Pull code
- Run `gulp jekyll`
- Check that http://127.0.0.1:4001/vanilla-brochure-theme/examples/patterns/muted-heading/ matches [pattern spec](https://github.com/ubuntudesign/vanilla-brochure-theme-design/tree/master/Muted%20heading) 

## Note

The spec doesn't say that this should be `text-align: center` but from previous experience, this pattern is alway centered so I've made it so here. Whoever does design review, please confirm.

## Details

Fixes #66
